packages/cli/src/playground/src/App.tsx

```diff
@@ -18,74 +18,77 @@ import WorkflowTracesPage from './pages/workflows/workflow/traces';
 import Networks from './pages/networks';
 import { NetworkLayout } from './domains/networks/network-layout';
 import Network from './pages/networks/network';
+import { PostHogProvider } from './lib/analytics';
 
 function App() {
   return (
-    <BrowserRouter>
-      <Routes>
-        <Route
-          element={
-            <Layout>
-              <Outlet />
-            </Layout>
-          }
-        >
-          <Route path="/networks" element={<Networks />} />
-          <Route path="/networks/:networkId" element={<Navigate to="/networks/:networkId/chat" />} />
+    <PostHogProvider>
```

Comment:
ℹ️ This is the moving bit, but since it's at the top level of the tree, everything shifts one line below 🙃